src/Combinatorics/Graphs/functions.jl

```diff
@@ -1744,3 +1744,37 @@ function maximal_cliques(g::Graph{Undirected})
     Polymake.call_function(:graph,:max_cliques,g.pm_graph)
   ))
 end
+
+@doc raw"""
+     is_acyclic(G::Graph{Directed})
+
+Determines if `G` is acyclic by performing topological sort.
+Uses Kahn's algorithm, starting with minimal elements.
+
+# Examples
+```jldoctest
+julia> g = graph_from_edges(Directed, [[1, 2], [2, 3], [1, 3], [2, 4], [3, 4]])
+Directed graph with 4 nodes and the following edges:
+(1, 2)(1, 3)(2, 3)(2, 4)(3, 4)
+
+julia> is_acylic(g)
+true
+```
+"""
+function is_acylic(G::Graph{Directed})
+  a = adjacency_matrix(G)'
+
+  # dont totally understand why this line works 
+  S = findall(map(isone, .!any(a; dims=2)'[:]))
```

Comment:
Unless I misunderstood it this should work as well:
```suggestion
  S = findall(isempty, Polymake.col.(Ref(a), 1:ncols(a)))
```
or
```suggestion
  S = findall(iszero, eachcol(a))
```
(more readable but less efficient)